### DISCUSSION TITLE
Tapi3.dll

### DISCUSSION THREAD

**Discussion by LotharHellwig** [community/community#129963] [Actions]
### Select Topic Area

Question

### Body

Hallo, ich kann die ITapi3.dll unter Windows11 (neuer Laptop) nicht registrieren

Wer kann Helfen?

Danke gruß Lothar

**Comment by Akash1134** [comment]
Hi @LotharHellwig ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Reply by LotharHellwig** [reply]
thanks, ihope
